reflex/utils/prerequisites.py

```diff
@@ -326,14 +327,22 @@ def install_node():
         # Add execute permissions to fnm executable.
         os.chmod(constants.FNM_EXE, stat.S_IXUSR)
         # Install node.
+        # Specify arm64 arch explicitly for M1s and M2s.
```

Comment:
Hello, what if I have an ARM chip computer and is not a Apple product?

Reply:
should already be handled for linux `aarch64`. windows also seems to work as is.

you can test with `main` and see if there is some error?

Reply:
Hello, thank you the reply. Are you refering to execute the test using a computer based on arm & Apple?, if so, sure I can, I have a M1 chip in hands.

Reply:
The reflex dev team mostly uses M1 and M2 macs, so we have pretty good coverage on that. I have an aarch64 linux laptop (pine book pro) that i've tested on, which works. And I have an armv7 (32-bit) raspberry pi, that I've been meaning to test with, but haven't yet.

If there is some particular os/arch combination that you're concerned about, my suggestion would be to test running `pip install git+https://github.com/reflex-dev/reflex` and `reflex init` your project and see if it's working. If not, please file a bug and we will do our best to accomodate it or steer you in the right direction. 

The assumption at this point is the new `fnm`-based node bootstrap is working on windows (x86, x64, arm64), macos (x64, arm64), and linux (x64, arm64, arm32).

Reply:
![image](https://github.com/reflex-dev/reflex/assets/12007227/80eb4822-2084-4927-b279-0ecec54ac975)
hello, I run ```pip install git+https://github.com/reflex-dev/reflex``` and ```reflex init```, but this bug still shows sometimes, I use mac M1

Reply:
I find that some error operations like index overflow or database error will break reflex down and can not re-open

Reply:
@ltm920716 did you remove the reflex dir (`rm -rf /User/<username>/Application Support/reflex`) before running `reflex init`

Reply:
sorry that there is no 'Application Support' dir in this ’/User/<username>/‘ path, I wiil try to find it，thanks

Reply:
I find that path is ```/Users/<username>/Library/Application Support```